docsite/build-site.py

```diff
@@ -69,10 +70,10 @@ def __init__(self):
 
             app.builder.build_all()
 
-        except ImportError, ie:
```

Comment:
The `ie` variable is unused, so I've opted to drop it rather than simply updating the syntax to `except ImportError as ie`.